client/src/components/Landing/WorkflowLanding.vue

```diff
@@ -21,21 +20,17 @@ const props = withDefaults(defineProps<Props>(), {
 const workflowId = ref<string | null>(null);
 const errorMessage = ref<string | null>(null);
 const requestState = ref<Record<string, never> | null>(null);
+const instance = ref<boolean>(false);
 
-onMounted(async () => {
-    const payload: ClaimLandingPayload = {};
-    if (props.secret) {
-        payload["client_secret"] = props.secret;
-    }
-
-    const { data, error } = await GalaxyApi().POST("/api/workflow_landings/{uuid}/claim", {
```

Comment:
I mean... the intended approach should still work right 😅 . You did something very different here it seems and that is fine but I think private claims should still work.

Reply:
xref https://github.com/mvdbeek/iwc/blob/website/website/pages/workflow/%5Bid%5D.vue#L56-L73 - is the idea here that it works with an already logged in Galaxy only and the request is made as the user? Why not give a link that requires a log in when you land on it? Is that hard to implement?

Reply:
> xref https://github.com/mvdbeek/iwc/blob/website/website/pages/workflow/%5Bid%5D.vue#L56-L73 - is the idea here that it works with an already logged in Galaxy only and the request is made as the user? 

No, the landing page creation can only be anonymous as far as I understand. Calls from external sites shouldn't send sessions or similar sensitive stuff.

> Why not give a link that requires a log in when you land on it?

That's what I'm doing ... right now you have to be logged in but the idea is to just embed the login / register component and move on once they're logged in.

Reply:
Or are you asking why I'm not claiming the landing request ? That's just cause navigating back and refreshing should work IMO.

Reply:
Will sharing work - you're creating the landing with a particular user in there - I feel like I had tests to ensure that is pretty explicitly tied to the user. I would call that a serious bug/oversight on my part if it works. Could we introduce something in the model to declare that landing is anonymous and works that way? I'd be happy to take over the PR and implement that - you already have a nice migration setup in the PR and I don't want to create more work for you.

Reply:
Sorry John, I don't see how the landing would be useful for BRC if we need an authenticated user. I don't think it's a bug. Isn't the client secret the way to make it private ?

Reply:
Discussed this on gitter and I think we're on the same page now, we need to separate private and public workflow landing requests

Reply:
Alright, both is possible now depending on the value of the `public` query parameter. If not provided we try to claim the request.